bloom-housing/bloom
# feat: lottery toggle and empty state

# Pull Request Template

## Issue Overview

This PR addresses #4045 #4046


## Description

This PR includes the partners frontend lottery toggle and creates the lottery page with its empty state and some mock data.

## How Can This Be Tested/Reviewed?

To view the lottery tab you need to have the partner toggle on, and visit a closed listing that has selected a lottery review order.

The lottery page itself should match the empty view in [this Figma screen](https://www.figma.com/file/PAeCVANDmGdTz2Y53ouErM/Lotteries?type=design&node-id=3235-29419&mode=design&t=srolHIDM18Eg99fe-0), with the addition of the "new" tag in the tab.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
